menu.js

```diff
@@ -19,6 +19,16 @@ function sendAction(action, data) {
 
 const viewSubmenu = [
 	{
+		label: 'Toggle Dark Mode',
```

Comment:
I think we should move it so About for consistency. It works nice in Caprine as it's next to the settings shortcut.